cve_bin_tool/checkers/glibc.py

```diff
@@ -22,6 +22,8 @@ class GlibcChecker(Checker):
         r"glibc",
         r"nscd",
         r"libc.so.",
+        r"subversion",
+        r"svn",
```

Comment:
I think these were put in here by mistake? they shouldn't be in the glibc checker.